pyccl/core.py

```diff
@@ -1148,3 +933,278 @@ def __init__(self, **kwargs):
                              "%s " % list(p.keys()))
         kwargs.update(p)
         super(CosmologyVanillaLCDM, self).__init__(**kwargs)
+
+
+class CosmologyCalculator(Cosmology):
```

Comment:
Why are we not allowing mu-Sigma MG for the calculator?

Reply:
My idea was that the calculator mode is intended for you to construct your own fancy non-standard cosmology from arrays. The main aim of the cosmological parameters needed to initialize it is that certain things (mostly the halo model) require knowledge of what Omega_M and h are. If you want to use CCL's mu-Sigma, isitgr etc., just use the standard CCL cosmology.

Reply:
Fair enough. FWIW, most of the prescriptions for LSS quantities that need these other parameters are calibrated assuming LCDM. Thus I think the main use for these params is in things like computing 2pt functions etc. This is fine OFC, just something for users to be mindful of.